lib/authlogic/session/password.rb

```diff
@@ -233,6 +234,14 @@ def password_field
           def verify_password_method
             self.class.verify_password_method
           end
+          
+          def parse_param_val(value)
```

Comment:
Please add a comment here explaining why this is necessary, e.g. the changes to AC::Parameters in rails 5 (http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#actioncontroller-parameters-no-longer-inherits-from-hashwithindifferentaccess)